### PR TITLE
fix: wizards copy

### PR DESCRIPTION
**Pull request by loks0n**

### CODE REVIEW

File: src/routes/console/(migration-wizard)/step1.svelte [hunk 1]
```diff
@@ -44,6 +44,7 @@
 
 <WizardStep {beforeSubmit}>
     <svelte:fragment slot="title">
+        <!-- @vincentge - What about this? -->
```
Comment (gewenyu99) [inline src/routes/console/(migration-wizard)/step1.svelte:47]: Project

File: src/routes/console/project-[project]/overview/platforms/wizard/android/step3.svelte [hunk 2]
```diff
@@ -15,6 +15,7 @@ val client = Client(context)
 </script>
 
 <WizardStep>
+    <!-- @vincentge - What about this? -->
     <svelte:fragment slot="title">Initialize SDK</svelte:fragment>
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/android/step3.svelte:19]: Initialize

File: src/routes/console/project-[project]/overview/platforms/wizard/apple/step2.svelte [hunk 3]
```diff
@@ -29,6 +29,7 @@
 </script>
 
 <WizardStep>
+    <!-- @vincentge - What about this? -->
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/apple/step2.svelte:32]: Install

File: src/routes/console/project-[project]/overview/platforms/wizard/apple/step3.svelte [hunk 4]
```diff
@@ -17,6 +17,7 @@ let client = Client()
 </script>
 
 <WizardStep>
+    <!-- @vincentge - What about this? -->
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/apple/step3.svelte:20]: Initialize

File: src/routes/console/project-[project]/overview/platforms/wizard/flutter/step2.svelte [hunk 5]
```diff
@@ -9,6 +9,7 @@
 </script>
 
 <WizardStep>
+    <!-- @vincentge - What about this? -->
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/flutter/step2.svelte:12]: See above examples

File: src/routes/console/project-[project]/overview/platforms/wizard/flutter/step3.svelte [hunk 6]
```diff
@@ -18,6 +18,7 @@ client
 </script>
 
 <WizardStep>
+    <!-- @vincentge - What about this? -->
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/flutter/step3.svelte:21]: Ditto

File: src/routes/console/project-[project]/overview/platforms/wizard/step4.svelte [hunk 7]
```diff
@@ -6,6 +6,7 @@
 </script>
 
 <WizardStep>
+    <!-- @vincentge - What about this? -->
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/step4.svelte:9]: All set

File: src/routes/console/wizard/cloudOrganizationChangeTier/usageExcess.svelte [hunk 8]
```diff
@@ -7,8 +7,9 @@
 </script>
 
 <WizardStep>
-    <svelte:fragment slot="title"
-        >Your usage exceeds the limits of the {tierToPlan($organization.billingPlan).name} plan</svelte:fragment>
+    <svelte:fragment slot="title">
+        <!-- @vincentge - What about this? -->
```
Comment (gewenyu99) [inline src/routes/console/wizard/cloudOrganizationChangeTier/usageExcess.svelte:11]: This is fine.

File: src/routes/console/wizard/feedback/step1.svelte [hunk 9]
```diff
@@ -32,6 +32,7 @@
 </script>
 
 <WizardStep {beforeSubmit}>
+    <!-- @vincentge - What about this? -->
```
Comment (gewenyu99) [inline src/routes/console/wizard/feedback/step1.svelte:35]: This is fine imo

File: src/routes/console/wizard/feedback/step2.svelte [hunk 10]
```diff
@@ -6,6 +6,7 @@
 </script>
 
 <WizardStep>
+    <!-- @vincentge - What about this? -->
```
Comment (gewenyu99) [inline src/routes/console/wizard/feedback/step2.svelte:9]: Thank you

File: src/routes/console/wizard/support/step1.svelte [hunk 11]
```diff
@@ -21,6 +21,7 @@
 </script>
 
 <WizardStep>
+    <!-- @vincentge - What about this? -->
```
Comment (gewenyu99) [inline src/routes/console/wizard/support/step1.svelte:24]: This is fine

**Review by gewenyu99 — APPROVED** [review]
Left suggestions

File: src/routes/console/wizard/feedback/step2.svelte [hunk 12]
```diff
@@ -6,7 +6,7 @@
 </script>
 
 <WizardStep>
-    <svelte:fragment slot="title">Thanks for the feedback!</svelte:fragment>
+    <svelte:fragment slot="title">Thanks you</svelte:fragment>
```
Comment (heyCarla) [inline src/routes/console/wizard/feedback/step2.svelte:9]:
```suggestion
    <svelte:fragment slot="title">Thank you</svelte:fragment>
```

**Review by heyCarla — CHANGES_REQUESTED** [review]
Just one typo. Otherwise LGTM!

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**